packages/s/shepherd.js.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "shepherd.js",
+  "description": "Guide your users through a tour of your app.",
+  "keywords": [
+    "site tour",
+    "tour",
+    "tutorial",
+    "shepherd"
+  ],
+  "authors": [
+    "Adam Schwartz <adam.flynn.schwartz@gmail.com>",
+    "Zack Bloom <zackbloom@gmail.com>"
+  ],
+  "maintainers": [
+    "Robert Wagner <rwwagner90@gmail.com>",
+    "Chuck Carpenter <chuck@shipshape.io>"
+  ],
+  "license": "MIT",
+  "homepage": "https://shepherdjs.dev",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/shipshapecode/shepherd.git"
+  },
+  "npmName": "shepherd.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.@(js|css|map)"
```

Comment:
```suggestion
        "**/*.@(js|css|map)"
```

It looks like the files are in `js` and `css` subdirs, so we need a recursive glob.

[Glob demo](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A%2A%2F%2A.%40%28js%7Ccss%7Cmap%29&matches=false&tests=%2F%2F%20Imported%20from%20shepherd.js%20NPM%20package&tests=.browserslistrc&tests=.eslintignore&tests=.nojekyll&tests=LICENSE&tests=dist%2Fcss%2Fshepherd.css&tests=stats.html&tests=.jsdoc.js&tests=.prettierrc.js&tests=src%2Fjs%2Futils%2Fauto-bind.js&tests=babel.config.js&tests=src%2Fjs%2Futils%2Fbind.js&tests=src%2Fjs%2Futils%2Fcleanup.js&tests=src%2Fjs%2Fevented.js&tests=src%2Fjs%2Futils%2Fgeneral.js&tests=jest.config.js&tests=src%2Fjs%2Futils%2Foverlay-path.js&tests=src%2Fjs%2Futils%2Fpopper-options.js&tests=rollup.config.js&tests=dist%2Fjs%2Fshepherd.esm.js&tests=dist%2Fjs%2Fshepherd.esm.min.js&tests=dist%2Fjs%2Fshepherd.js&tests=src%2Fjs%2Fshepherd.js&tests=dist%2Fjs%2Fshepherd.min.js&tests=src%2Fjs%2Fstep.js&tests=svelte.config.js&tests=tailwind.config.js&tests=src%2Fjs%2Ftour.js&tests=src%2Fjs%2Futils%2Ftype-check.js&tests=docs-src%2Ftutorials%2Fconfig.json&tests=cypress.json&tests=package.json&tests=dist%2Fjs%2Fshepherd.esm.js.map&tests=dist%2Fjs%2Fshepherd.esm.min.js.map&tests=dist%2Fjs%2Fshepherd.js.map&tests=dist%2Fjs%2Fshepherd.min.js.map&tests=docs-src%2Ftutorials%2F01-install.md&tests=docs-src%2Ftutorials%2F02-usage.md&tests=docs-src%2Ftutorials%2F03-styling.md&tests=docs-src%2Ftutorials%2F04-cookbook.md&tests=CHANGELOG.md&tests=HISTORY.md&tests=README.md&tests=patches%2Frollup-plugin-eslint%2B7.0.0.patch&tests=src%2Fjs%2Fcomponents%2Fshepherd-button.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-cancel-icon.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-content.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-element.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-footer.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-header.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-modal.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-text.svelte&tests=src%2Fjs%2Fcomponents%2Fshepherd-title.svelte&tests=src%2Ftypes%2Fevented.d.ts&tests=src%2Ftypes%2Fshepherd.d.ts&tests=src%2Ftypes%2Fstep.d.ts&tests=src%2Ftypes%2Ftour.d.ts&tests=.dependabot%2Fconfig.yml&tests=.github%2Fworkflows%2Fpublish-github-pages.yml&tests=.github%2Fstale.yml)